src/apps/frontend/pages/authentication/login/login-form.tsx

```diff
@@ -0,0 +1,84 @@
+/* eslint-disable @typescript-eslint/restrict-template-expressions */
+import { Block } from 'baseui/block';
+import { FormControl } from 'baseui/form-control';
+import { Input } from 'baseui/input';
+import { toaster } from 'baseui/toast';
+import { HeadingMedium } from 'baseui/typography';
+import React, { useEffect } from 'react';
+
+import {
+  Button,
+} from '../../../components';
+
+import useLoginForm from './login-form.hook';
+
+export default function LoginForm(): React.ReactElement {
+  const {
+    formik, isLoading, error, result,
```

Comment:
`form`

Later, it should probably be used as:

`<form {...form}>`

Reply:
Done